inc/classes/settings/class-secupress-settings.php

```diff
@@ -326,6 +332,8 @@ protected function do_sections() {
 		 */
 		do_action( 'secupress.settings.before_section_' . $this->sectionnow, $with_save_button );
 
+		$this->section_is_disabled = true;
```

Comment:
So I'm missing something. On line 335 the prop is set to "true" and on line 349 we check it. How this prop could be "false"?